x-pack/plugins/session_view/common/utils/sort_processes.test.ts

```diff
@@ -23,7 +23,8 @@ describe('sortProcesses(a, b)', () => {
       const current = processes[i];
       const next = processes[i + 1];
       expect(
-        new Date(next.getDetails().process.start) >= new Date(current.getDetails().process.start)
+        new Date(next.getDetails().process!.start!) >=
```

Comment:
Hey, I'm wondering if it's really correct to be `process!.start!`? 

according with the [docs](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-):

> it’s important to only use `!` when you know that the value can’t be `null` or `undefined.

which I think it's the opposite in this case, and should be instead `new Date(next.getDetails()?.process?.start)`

Reply:
Yea that's correct, I only did `!` for tests related stuff haha